style.css

```
@@ -0,0 +1,33 @@
+.morning {
+    text-align: center;
+}
+
+img {
+    max-width: 60%;
+    position: relative;
+    align-content: center;
+}
+
+
+.full {
+    max-width: 100%;
```

Same note about using `width` vs `max-width`:
```suggestion
    width: 100%;
```